### ISSUE TITLE
Sales person  should  have inventory alert on dashboard

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa#40]

**Comment by lcabrini** [comment]
If this is a user story it should be written as one. If it is not, it should not be in the "user stories" project.

If is a design task, and there is a proposed design, can we see it? I have not seen any proposal for a design.

**Comment by BorderITS** [comment]
is something like this okay ?

**Comment by lcabrini** [comment]
It is still in two different projects at the same time. That is not possible. Either an issue is a user story or it is a UI design task. NEVER the two.

**Comment by BorderITS** [comment]
Done and thanks for your correction

**Comment by lcabrini** [comment]
This looks quite good now, except for the dashboard part. Mostly (although there may be a few exceptions) user stories are not concerned with how information is presented. That would more be a design issue.

We may well have this information on the dashboard. But you could easily imagine other delivery methods for stock alerts: email, SMS, etc.

So, it is better for the user story to less opinionated about how information will ultimately be present to the user. I would suggest "as a sales person I want to receive stock alert notifications".

**Comment by lcabrini** [comment]
Also, I wonder if this shouldn't be relevant also for the warehouse manager? I'm not sure what we do here. Let's do some research on stories that have more than one role.

**Comment by BorderITS** [comment]
ok

**Comment by lcabrini** [comment]
In the meantime can we remove the on the dashboard part as per my previous comment?

**Comment by BorderITS** [comment]
Ok so do you want me to rename it?

**Comment by lcabrini** [comment]
Yes, let's stick to the story I suggested above, unless you have any other ideas?

**Comment by BorderITS** [comment]
but the client was saying that if the sales person can have an alert on their dashboard that's why i raised that issue

**Comment by lcabrini** [comment]
I understand, and they can have it. But, the user story is mostly about capturing required functionality. Once the system is able to raise stock alers, we can then worry about where those alerts should appear. The latter part is a UI issue.

Moreover, this is a user story. Move it out of UI and into user stories.

**Comment by BorderITS** [comment]
I have made the changes
